app/assets/javascripts/oxalis/view/layouting/golden_layout_adapter.js

```diff
@@ -46,7 +36,10 @@ const monkeypatchGLSizeGetter = gl => {
     if (value) {
       return _oldWidth.call(gl, value);
     } else {
-      const { width } = getGroundTruthLayoutRect();
+      let { width } = getGroundTruthLayoutRect();
+      if (width === undefined) {
+        width = 500;
```

Comment:
Now, the `500` is scattered around in this file. Can you extract it into a `minimumExtent` variable?